.evergreen/config.yml

```diff
@@ -1983,6 +1983,7 @@ buildvariants:
     os:
       - ubuntu-20.04
     mongodb-version:
+      - "latest"
```

Comment:
The updated tests can only run on 7.0+, which the csfle suite wasn't running on before.  `latest` as of yesterday was `"7.1.0-alpha-220-g77c517c"` so that covers it, and it seems good to be running csfle tests against `latest` anyway.